charts/zookeeper-operator/templates/clusterrole.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.rbac.create }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "zookeeper-operator.fullname" . }}
+rules:
+- apiGroups:
+  - zookeeper.pravega.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
```

Comment:
Same as earlier helm test. Can we not do with only `get` under 'verbs` here?

Reply:
This piece of code is for granting permissions for zookeeper operator deployment and not for running the test.